File: app/src/main/java/com/blog/demo/control/edit/EditTextImeOptionsActivity.java
package com.blog.demo.control.edit;

import android.app.Activity;
import android.os.Bundle;
import android.view.KeyEvent;
import android.view.inputmethod.EditorInfo;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import androidx.annotation.Nullable;

import com.blog.demo.R;

public class EditTextImeOptionsActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_control_edit_text_ime_options);

        EditText editText = findViewById(R.id.edit_text);
        editText.setOnEditorActionListener(new TextView.OnEditorActionListener() {
            @Override
            public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
                if (actionId == EditorInfo.IME_ACTION_NEXT) {
                    Toast.makeText(EditTextImeOptionsActivity.this, "Next Click",
                            Toast.LENGTH_LONG).show();
                    return true;
                }
                return false;
            }
        });
    }

}

File: app/src/main/java/com/blog/demo/control/edit/EditTextImeOptionsActivity.kt
package com.blog.demo.control.edit

import android.app.Activity
import android.os.Bundle
import android.view.inputmethod.EditorInfo
import android.widget.EditText
import android.widget.Toast
import com.blog.demo.R

class EditTextImeOptionsActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_control_edit_text_ime_options)

        val editText: EditText = findViewById(R.id.edit_text)
        editText.setOnEditorActionListener { _, actionId, _ ->
            if (actionId == EditorInfo.IME_ACTION_NEXT) {
                Toast.makeText(this@EditTextImeOptionsActivity, "Next Click", Toast.LENGTH_LONG).show()
                true
            }
            false
        }
    }

}